src/CitMovie.Business/Managers/PersonManager.cs

```diff
@@ -12,17 +12,24 @@ public PersonManager(IPersonRepository personRepository, IMapper mapper)
     }
 
 
-    public async Task<IEnumerable<PersonResult>> GetPersonsAsync(int page, int pageSize)
-
+    public async Task<IEnumerable<PersonResult>> QueryPersonsAsync(PersonQueryParameter queryParameter, PageQueryParameter pageQuery, int? userId)

```

Comment:
Maybe we should have some null check for the queryParameter. The QueryPerson endpoint does not check if queryParameter is null before using it. I feel like we should make this check one of the places

Reply:
This is not a problem, the aspnet framework initializes a new instance of the objects regardless, if you look at the properties in the `PersonQueryParameter ` and `PageQueryParameter ` they have been specified as nullable or not.